msl_robot/src/robotmovement/RobotMovement.cpp

```diff
@@ -624,6 +624,32 @@ void RobotMovement::clearPTControllerQueues()
     }
 }
 
+void RobotMovement::updatePT()
+{
+    auto odom = this->wm->rawSensorData->getOwnVelocityMotion();
+    auto mc = this->wm->rawSensorData->getLastMotionCommand();
+    if (odom != nullptr)
```

Comment:
This would probably be nicer and reduce indentation:

```
if (odom == nullptr) {
    initializePTControllerParameters();
    return;
}

clearPTControllerQueues();
...
```